libs/java/dal/db/scripts/setup.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-# Import the private.key that is in the root folder.
-# This is required to sign and verify packages.
-gpg --import private.key
-
 # This fixes a bug with GPG when signing the package.
-export GPG_TTY=$(tty)
-export MAVEN_OPTS="--add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.desktop/java.awt.font=ALL-UNNAMED"
+# export GPG_TTY=$(tty)
+# export MAVEN_OPTS="--add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.desktop/java.awt.font=ALL-UNNAMED"
 
+# Import the private.key that is in the root folder.
+# This is required to sign and verify packages.
+# sudo env "PATH=$PATH" gpg --import private.key
+sudo gpg --import private.key
```

Comment:
Required `sudo` to import the key.  For some reason some containers won't work without.  I don't understand why.